packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -336,6 +341,39 @@ abstract class VpcBase extends Resource implements IVpc {
     };
   }
 
+  /**
+   * Adds a VPN Gateway to this VPC
+   */
+  public enableVpnGateway(options: VpnGatewayProps): void {
+    if (this.vpnGatewayId) {
+      // already enabled
+      return;
+    }
+    const vpnGateway = new CfnVPNGateway(this, 'VpnGateway', {
```

Comment:
Why did you define `class VpnGateway` if you're not going to use it here?

Reply:
The gatewayId is being used now: `this._vpnGatewayId = vpnGateway.ref;`

Reply:
Sure, but you are instantiating a `CfnVPNGateway`. Should you not be instantiating the `VpnGateway` here?

Reply:
Good catch!!!. Done